freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/image/Image.java

```diff
@@ -32,6 +34,8 @@
 
     private final Map<String, String> packageVersions;
 
+    private boolean advertised;
+
     @JsonCreator
     public Image(
```

Comment:
Totally optional and I'll leave this to you, but the property declarations in the class look messy altogether:
 - some have their names as constants, others don't
 - some have the JsonProperty annotation on their getter method as well, others don't
 - constants have PROPERTY in their names, but the new ADVERTISED doesn't
 I think cleaning this up is a low hanging fruit.